utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -0,0 +1,244 @@
+  
+package com.esri.samples.display_utility_associations;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.UniqueValueRenderer;
+import com.esri.arcgisruntime.utilitynetworks.UtilityAssociation;
+import com.esri.arcgisruntime.utilitynetworks.UtilityAssociationType;
+import com.esri.arcgisruntime.utilitynetworks.UtilityNetwork;
+import com.esri.arcgisruntime.utilitynetworks.UtilityNetworkSource;
+
+public class DisplayUtilityAssociationsSample extends Application {
+  
+  private MapView mapView;
+  private GraphicsOverlay associationsOverlay;
+  private UtilityNetwork utilityNetwork;
+  
+  @Override
+  public void start(Stage stage) {
+    
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      
+      // set title, size, and add scene to stage
+      stage.setTitle("Display Utility Association Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+  
+      // create image views for the association symbols to display them in the legend
+      ImageView attachmentImageView = new ImageView();
+      ImageView connectivityImageView = new ImageView();
+  
+      // create Labels for the preview
+      Label attachmentLabel = new Label("Attachment");
+      Label connectivityLabel = new Label("Connectivity");
+  
+      // create Symbols for the associations and preview
+      SimpleLineSymbol attachmentSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, 0xFF00FF00, 5);
+      SimpleLineSymbol connectivitySymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, 0xFFFF0000, 5);
+  
+      // add the attachment symbol to the image view to display it in the legend
+      ListenableFuture<Image> attachmentImage  = attachmentSymbol.createSwatchAsync(0x00000000);
+      attachmentImage.addDoneListener(() -> {
+        try {
+          // display the image view in the preview area
```

Comment:
I think technically we're setting the `Image` to the `ImageView`, so we can try and be more specific in the comment:
```suggestion
          // display the image in the image view
```